phc/easy/document_reference.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+from phc.easy.frame import Frame
+from phc.easy.patient_item import PatientItem
+
+
+class DocumentReference(PatientItem):
+    @staticmethod
+    def table_name():
+        return "document_reference"
+
+    @staticmethod
+    def code_keys():
+        return ["type.coding", "content.attachment", "meta.tag"]
+
+    @staticmethod
+    def transform_results(df: pd.DataFrame, **expand_args):
+        return Frame.expand(
+            df,
+            code_columns=[*expand_args.get("code_columns", []), "type"],
+            custom_columns=[
+                *expand_args.get("custom_columns", []),
+                Frame.codeable_like_column_expander("subject"),
+                # TODO: Properly parse content column
```

Comment:
Just making sure this is something you're intentionally leaving for later as opposed to something todo before this is merged.

Reply:
That is correct.